packages/cypress-commands/src/setups/enableNetworkShim/stubRequests.js

```diff
@@ -0,0 +1,60 @@
+import {
+    splitHostAndPath,
+    getFullTestName,
+    findMatchingRequestStub,
+    isPathStaticResource,
+} from './utils'
+
+export default function stubRequests(state) {
+    state.config.hosts.forEach(host => {
+        cy.intercept(host, request => {
+            try {
+                stubRequest(state, request)
+            } catch (error) {
+                console.error('NetworkShim stub error', error)
+            }
+        })
+    })
+}
+
+function stubRequest(state, request) {
```

Comment:
What happens if changes caused there to be a new, not yet captures, request? It seems like this case is not covered